mlir/test/Dialect/ArmSME/canonicalize.mlir

```diff
@@ -1,18 +1,16 @@
 // RUN: mlir-opt -canonicalize -split-input-file -verify-diagnostics %s | mlir-opt | FileCheck %s
 
-// This tests that the `arm_sme.materialize_ssa_tile` placeholder is removed
-// once it becomes unused, after lowering to control flow.
+// This tests that dead tile values are removed from control flow.
 
 // -----
```

Comment:
unrelated but this should come after the first test, in this case I think `-split-input-file` can be removed altogether given there's a single test.